src/components/Preferences/Preferences.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Transition, animated } from 'react-spring'
+import {
+  AppBar,
+  AppView,
+  BREAKPOINTS,
+  ButtonIcon,
+  IconClose,
+  TabBar,
+  Viewport,
+  breakpoint,
+  font,
+  springs,
+} from '@aragon/ui'
+import CustomLabels from './CustomLabels'
+
+const SMALL_TABS = ['Network', 'Manage labels']
+
+const TABS = ['Network', 'Addresses', 'Manage labels']
+
+const Preferences = ({ onClose, smallView }) => {
+  const [selectedTab, setSelectedTab] = React.useState(0 || smallView ? 1 : 2)
+
+  return (
+    <AppView
+      smallView={smallView}
+      padding={0}
+      appBar={
+        <StyledAppBar>
+          <Title>My Preferences</Title>
+          <StyledButton label="Close" onClick={onClose}>
+            <IconClose />
+          </StyledButton>
+        </StyledAppBar>
+      }
+    >
+      <Section>
+        <TabBar
+          items={smallView ? SMALL_TABS : TABS}
+          selected={selectedTab}
+          onChange={setSelectedTab}
+        />
+        <Content>
+          {selectedTab === 0 && <ComingSoon />}
+          {!smallView && selectedTab === 1 && <ComingSoon />}
+          {((!smallView && selectedTab === 2) ||
+            (smallView && selectedTab === 1)) && <CustomLabels />}
```

Comment:
Can’t wait for [pattern matching](https://github.com/tc39/proposal-pattern-matching) in JS 😄

Reply:
I'll most probably take this out and just keep two tabs until the `Addresses` tab is implemented